db.js

```diff
@@ -392,18 +400,12 @@ exports.init = function (sbot, config) {
       }
     })
 
-    if (rest.length === 0) {
-      const ops = fromDB(jitdb)
-      ops.meta.db = this
-      return jitdbOperators.query(ops, and(waitUntilReady), first)
-    }
-
-    if (!first.meta) {
+    if (first.meta) {
+      return jitdbOperators.query(first, where(waitUntilReady), ...rest)
+    } else {
       const ops = fromDB(jitdb)
       ops.meta.db = this
-      return jitdbOperators.query(ops, and(waitUntilReady, first), ...rest)
-    } else {
-      return jitdbOperators.query(first, and(waitUntilReady), ...rest)
+      return jitdbOperators.query(ops, where(waitUntilReady), first, ...rest)
```

Comment:
At first reading this, I thought this needed a `where(and(waitUntilReady, first)` but now I see that doesn't work.

Reply:
Yeah, `first` could be `live` and `live` doesn't work inside `and` anymore (it used to).